Umbraco-Cloud/Set-Up/Dedicated/index.md

```diff
@@ -0,0 +1,55 @@
+---
+versionFrom: 8.0.0
+versionTo: 10.0.0
+---
+
+# Move your Umbraco Cloud project to dedicated resources
+
+In this article, you can read about how you enable your Umbraco Cloud project to run on dedicated resources and what you need to be aware of before you do so.
+
+## Before you move your project to dedicated resources
+
+Before you decide to move your Umbraco Cloud project, you need to consider a few things:
+
+- Currently, Umbraco Cloud offers dedicated resources for professional plans only. We expect to include dedicated resources for Starter and Standard plans in the self-service feature in Umbraco Cloud portal later in 2022.
+
+- In case you want to move your Umbraco Cloud project with a Starter or Standard project plan to a dedicated server in Umbraco Cloud, please reach out to our support who will assist you in doing this.
+
+- Moving from a shared plan to a dedicated plan will change the outgoing IP of the project. If your solution has an external service that requires whitelisting the outgoing IP of the project, we advise you to enable the static outbound IP feature for your cloud project and share that static outbound IP address with the third party. The static outbound IP address will not change when moving from a shared plan to a dedicated resource. For more info on static lease visit the documentation for [external services](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/External-Services/).
```

Comment:
```suggestion
- Moving from a shared plan to a dedicated plan will change the outgoing IP of the project. If your solution has an external service that requires whitelisting the outgoing IP, we advise you to enable the static outbound IP feature for your project and share that static outbound IP address with the third party. The static outbound IP address will not change when moving from a shared plan to a dedicated resource. For more info on static lease visit the documentation for [external services](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/External-Services/).
```

Reply:
Also, "static lease" - what does that mean exactly?
Is that the static IP? It's important to be consistent with the wording, so is there any chance it would make sense to say something like:
"For more info on static IPs, visit the documentation ... " ?

Reply:
Hmm, seems like some text unintentionally was deleted in my PR. I believe the wording should be "For more info on static outbound IP please...". How can I change this while this PR is still active? :-)